src/systemcmds/tests/test_microbench_uorb.cpp

```diff
@@ -171,4 +180,32 @@ bool MicroBenchORB::time_px4_uorb()
 	return true;
 }
 
+bool MicroBenchORB::time_px4_uorb2()
```

Comment:
What's different in `2`? What about giving it a name instead of 2?

Reply:
updated